erpnext/accounts/doctype/bank_transaction/bank_transaction.py

```diff
@@ -8,8 +8,23 @@
 
 
 class BankTransaction(StatusUpdater):
+	_cy_precision = None
```

Comment:
```suggestion
	_currency_precision = None
```

I don't like guessing